scalastyle-config.xml

```diff
@@ -1,11 +1,6 @@
 <scalastyle commentFilter="enabled">
  <name>Mesosphere Marathon configuration</name>
  <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"/>
- <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
```

Comment:
I agree with @jdef this shouldn't be touched. Should I revert it or keep it consistent with 1.4?